src/steps/video-setup/preview.tsx

```diff
@@ -145,3 +149,51 @@ const PreviewVideo: React.FC<{ input: Input }> = ({ input }) => {
     </div>
   );
 };
+
+export const DisplayAudioInfo: React.FC<{ stream: MediaStream }> = ({ stream }) => {
+  const hasAudio = stream.getAudioTracks().length;
+
+  return (
+    <div css={{
+      position: "absolute",
+      top: 8,
+      right: 8,
+      whiteSpace: "pre-line",
```

Comment:
I think this can be removed. And then the whole `css` attr can be one line presumably